tests/cypress/integration/e2e/collection.spec.js

```diff
@@ -0,0 +1,138 @@
+/* eslint-disable indent */
+
+/* eslint-disable no-undef */
+const baseUrl = 'http://localhost:9090'
+describe('Collection Page should display the various collections and their details.', () => {
+  beforeEach(() => {
+    cy.viewport(1280, 800)
+    Cypress.Cookies.defaults({
+      preserve: 'session_id',
+    })
+    Cypress.on('uncaught:exception', (err, runnable) => {
+      // returning false here prevents Cypress from
+      // failing the test
+      return false
+    })
+    // cy.wait(2000)
+  })
+
+  it('Navigate to the Collection Page', () => {
+    cy.viewport(1280, 800)
+    cy.visit(baseUrl + '/rmrk/collections')
+  })
+
+  context('Filter by New First', () => {
+    it('Should display the various collections and their details.', () => {
+      cy.get('.is-grouped > :nth-child(1) > .control > .select > select')
+        .then(($select) => {
+          const opt = $select.find('option[value="BLOCK_NUMBER_DESC"]')
+          $select.val(opt.attr('value'))
+          return $select
+        })
+        .trigger('change')
+    })
+    it('Validate Page loaded and view Newest first', () => {
+      cy.get('.columns').should('be.visible')
+      cy.wait(2000)
```

Comment:
try to get yourself familiar with the concept of intercepts https://docs.cypress.io/api/commands/intercept
`cy.wait()` is not the correct way to wait for data to be loaded
querying `.columns` is also not very clever, as it is a standard class of buefy and therefore used often throughout the platform. here it probably just works because it is the first occurrence to be found. furthermore, it doesnt tell us anything about the state of the page.